### DISCUSSION TITLE
Display Logbook section for template sensors in more-info dialog

### DISCUSSION THREAD

**Discussion by peledzius** [home-assistant/frontend#12167] [Dashboard]
Current situation in more-info dialog for template binary sensors:

![image](https://user-images.githubusercontent.com/24360494/160912122-9741883a-d7f7-4a6e-8b72-8abda7d59524.png)

It would make sense to show Logbook section for template sensors aswell:

![image](https://user-images.githubusercontent.com/24360494/160912424-5d5aa74a-3bef-4e30-ba4e-2349b4702009.png)

**Comment by ildar170975** [comment]
Possibly this should be available for sensors w/o UoM.

**Comment by bilalhp** [comment]
What is the way to convert this discussion to a feature request?

**Reply by ildar170975** [reply]
It seems to be already a FR.

![image](https://github.com/home-assistant/frontend/assets/71872483/a78cc281-4cb3-48a0-99ca-f29beb58d04b)

**Reply by bilalhp** [reply]
...and it is being ignored or what?

**Comment by ildar170975** [comment]
https://github.com/home-assistant/frontend/issues/18847

**Comment by JCalvi** [comment]
I vote for this also.
